bahmnicore-api/src/main/java/org/bahmni/module/bahmnicore/service/impl/SMSServiceImpl.java

```diff
@@ -0,0 +1,77 @@
+package org.bahmni.module.bahmnicore.service.impl;
+
+import org.apache.commons.lang.StringUtils;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import org.bahmni.module.bahmnicore.contract.SMS.SMSRequest;
+import org.bahmni.module.bahmnicore.service.BahmniDrugOrderService;
+import org.bahmni.module.bahmnicore.service.SMSService;
+import org.openmrs.Patient;
+
+import org.openmrs.api.context.Context;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.text.MessageFormat;
+import java.util.Date;
+import java.util.Locale;
+
+import static org.bahmni.module.bahmnicore.util.BahmniDateUtil.convertUTCToGivenFormat;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.apache.http.HttpResponse;
+import org.apache.http.client.HttpClient;
+import org.apache.http.client.methods.HttpPost;
+import org.apache.http.entity.StringEntity;
+import org.apache.http.impl.client.HttpClientBuilder;
+
+@Service
+public class SMSServiceImpl implements SMSService {
+    private static Logger logger = LogManager.getLogger(BahmniDrugOrderService.class);
+    private final static String PRESCRIPTION_SMS_TEMPLATE = "bahmni.prescriptionSMSTemplate";
+    private final static String SMS_TIMEZONE = "bahmni.sms.timezone";
+    private final static String SMS_URL = "bahmni.sms.url";
+
+    @Autowired
```

Comment:
you dont need autowired annotation here, do you?

Reply:
yes, removed the annotation